src/app/routes/utils/regex/utils/index.js

```diff
@@ -75,3 +75,8 @@ export const getMostReadDataRegex = services => {
   const serviceRegex = getServiceRegex(services);
   return `/:service(${serviceRegex})/mostread:variant(${variantRegex})?.json`;
 };
+
+export const getSecondaryColumnDataRegex = services => {
+  const serviceRegex = getServiceRegex(services);
+  return `/:service(${serviceRegex})/sty-secondary-column:variant(${variantRegex})?.json`;
```

Comment:
We can remove the `.json` from here which allows us to remove the replacing of .json where we use this function.

`.json` gets added automatically with the fetchPageData function.

Reply:
Hmm seems to give a 404 when I change this. Don't we need this because this is the regex the express server uses?

Reply:
Sorry, I left this comment on the wrong method.

`getSecondaryColumnUrl ` should not need the `.json` extension.